package-lock.json

```diff
@@ -1,23 +1,8 @@
 {
   "name": "kafka-tutorials",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@bpurkaystha @danshafie I might have missed a comment about this, but just checking that the change to this file is intended.

Reply:
It appears that I've had a different node version when installing the packages, but that should not be too much of a concern. Thanks @ybyzek

Reply:
Since this isn't required for the PR, can you please back out these changes?

Reply:
Actually, in my opinion, this should be present there in the PR. Ideally, we'd want to ensure possiblity of time-travel of the past states. That's why we should keep it included in the PR. Moreover, this lock file contributes to the performance improvement by reducing the need to read `package.json` itself. It should be there in the version control.

Reply:
@bpurkaystha if the file should be updated, can we have this PR just about the breadcrumb and you can submit a separate PR for `package-lock.json`?

Reply:
Updated..!

Reply:
I agree the lockfile change isn't a big deal. Looks like the one in master was generated by a newer version of npm (>= 7) compared to the one in the referenced commit.

Since the project doesn't specify a semver for the npm version in the package.json this sort of change is expected and should be harmless. I think the only side effect is churn in this file. If one PR changes the lockfileVersion to `1`, and a developer pulls the changes, then runs `npm install` with a newer version of npm, there would be new changes to the lockfile. If those changes were committed, then pulled to an environment with an older npm version, running `npm install` would flip the version back. And so forth.

For this project and its relatively light set of node dependencies this will probably never matter, but if you want to avoid the flip-flopping, you can set something like `"engines": { "npm": ">=7.0.0"}` in the package.json. This should require an npm that uses the new lockfile version and avoid the flip-flop.

Reply:
@colinhicks https://github.com/confluentinc/kafka-tutorials/issues/1093 tracks your comment

@bpurkaystha thanks for backing out the change